docs/_index.md

```diff
@@ -3,6 +3,54 @@ title: "Welcome to Knative"
 linkTitle: "Documentation"
 weight: 10
 type: "docs"
+aliases:
+   - /docs/concepts/overview.md
 ---
 
-{{% readfile file="README.md" %}}
+The Knative project provides a set of [Kubernetes controllers](https://kubernetes.io/docs/concepts/architecture/controller/) that help to simplify developer and administrator experiences on [Kubernetes](https://kubernetes.io) clusters, by introducing an event-driven, serverless architecture.
+
+There are two core Knative components, that can be installed and used together or independently to provide different functions:
+
+* [Knative Serving](https://knative.dev/docs/serving/): Run stateless services more easily on Kubernetes,
+   by making autoscaling, networking and rollouts easier.
+
+* [Knative Eventing](https://knative.dev/docs/eventing/): Create subscriptions to event sources declaratively,
+   and route events to Kubernetes endpoints.
+
+These components are delivered as Kubernetes custom resource definitions (CRDs), which can be configured by a cluster administrator to provide default settings for developer-created applications and event workflow components.
+
+**Note**: Earlier versions of Knative included a Build component.  That component has been broken out into its own
+project, [Tekton Pipelines](https://tekton.dev/).
+
+Knative registers its own API types to the Kubernetes API, enabling developers and cluster administrators to use familiar languages and frameworks to solve common use cases, such as:
```

Comment:
This sentence feels "wrong" to me, because the first half doesn't really imply the second half.  What about:

> Knative builds higher-level APIs atop the existing Kubernetes API; Knative resources can interact easily with other Kubernetes-native resources when needed. Cluster Administrators will find that existing Kubernetes tools can be used to manage Knative as well. Developers will find that common languages and frameworks which include Kubernetes-friendly tooling work smoothly with Knative and reduce the time spent solving common deployment issues:

Reply:
I am assuming you are talking about Line 25 and on? 

I like where you are going with this, what about: 

> Knative builds more developer-friendly APIs atop the existing Kubernetes API so Knative resources can interact easily with other Kubernetes-native resources when needed. 

>Cluster Administrators will find that existing Kubernetes tools can be used to manage Knative as well. Developers will find that common languages and frameworks which include Kubernetes-friendly tooling work smoothly with Knative and reduce the time spent solving common deployment issues.

Reply:
@abrennan89 ^^^

Reply:
I'm going to do some editing myself on this part in the new PR and you guys can take a look. I'm not sold on either of these entirely because I think there's too much fluff in both that doesn't add value, but I get what you're trying to say.